WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift

```diff
@@ -230,6 +236,37 @@ extension OrdersRootViewController: OrderListViewControllerDelegate {
     }
 }
 
+// MARK: - Stored Order Settings (eg. filters)
+private extension OrdersRootViewController {
+    /// Fetch local Orders Settings (eg.  status or date range filters stored in Orders settings)
+    ///
+    func syncLocalOrdersSettings(onCompletion: @escaping (Result<StoredOrderSettings.Setting, Error>) -> Void) {
```

Comment:
This is still not used, but since we should integrate new order settings in the future, it will be useful to know the status, and in this way we use the same approach for the 3 methods for syncing, saving and resetting the settings, like we already did with Products.